Sources/Sentry/Public/SentrySpanContext.h

```diff
@@ -91,6 +114,23 @@ SENTRY_NO_INIT
                       operation:(NSString *)operation
                         sampled:(SentrySampleDecision)sampled;
 
+/**
```

Comment:
Ssame as in https://github.com/getsentry/sentry-cocoa/pull/4751#discussion_r1930846919